sensing/gnss_poser/CMakeLists.txt

```diff
@@ -39,3 +40,5 @@ ament_auto_package(INSTALL_TO_SHARE
   launch
   config
 )
+
+#ament_target_dependencies(gnss_poser_node PUBLIC "autoware_sensing_msgs")
```

Comment:
If you do not need this line, please delete it.

Reply:
Fixed! https://github.com/autowarefoundation/autoware.universe/pull/1230/commits/e7757d4f7d890ba721436139b4cc32ca82f77d81